carbon-location.html

```diff
@@ -28,6 +28,13 @@
       path: '/elements/carbon-route-converter'
     }
 
+The following queryParams object will also be produced:
```

Comment:
I think that we don't need to talk about queryParams in carbon-location as they're not part of its public API. WDYT?